Src/BootCamp.Chapter/Csv/CsvDelimiter.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace BootCamp.Chapter.Csv
+{
+    public enum CsvDelimiter
```

Comment:
Sometimes it's better to just use a constant instead of enum if we want to get a value. It's not a tag, it's rather a named constant.